products/amazon-linux.md

```diff
@@ -24,8 +24,8 @@ releases:
 -   releaseCycle: '2'
     releaseLabel: 'Amazon Linux 2'
     eol: 2025-06-30
-    latest: "2.0.20221210.0"
-    latestReleaseDate: 2022-12-16
+    latest: "2.0.20230119.1"
```

Comment:
There is an issue with the release link (due to `changelogTemplate` ?)

Reply:
#2397

Reply:
Filed https://github.com/endoflife-date/endoflife.date/pull/2398